src/main/java/pl/betoncraft/betonquest/objectives/PickupObjective.java

```diff
@@ -0,0 +1,125 @@
+package pl.betoncraft.betonquest.objectives;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Listener;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.EventHandler;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.event.entity.EntityPickupItemEvent;
+
+import pl.betoncraft.betonquest.BetonQuest;
+import pl.betoncraft.betonquest.Instruction;
+import pl.betoncraft.betonquest.api.Objective;
+import pl.betoncraft.betonquest.config.Config;
+import pl.betoncraft.betonquest.Instruction.Item;
+import pl.betoncraft.betonquest.utils.PlayerConverter;
+import pl.betoncraft.betonquest.exceptions.InstructionParseException;
+
+public class PickupObjective extends Objective implements Listener {
+
+    private final Item[] pickupItems;
+
+    private final int amount;
+
+    private final boolean notify;
+
+    public PickupObjective(Instruction instruction) throws InstructionParseException {
+        super(instruction);
+
+        template = PickupData.class;
+
+        pickupItems = instruction.getItemList();
```

Comment:
if i look at this line of code: https://github.com/BetonQuest/BetonQuest/blob/97e1ddc6106288fa5449c4e01f1b7fcd53efb69e/src/main/java/pl/betoncraft/betonquest/Instruction.java#L190
i see, that this allows to make a item list like that:
`emerald:5,dirt:30,diamonds:10`
it is realy cool that it is possible to specify multible items that can compleate the objective. But the amount is ignored. Maybe we want to use this amount!? idk